plugins/become/sudosu.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2021, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = """
+    become: sudosu
+    short_description: Run tashks using sudo su -
+    description:
+        - This become plugins allows your remote/login user to execute commands as another user via the sudo and su utility combined.
+    author: ansible (@core)
```

Comment:
The poor person called `core` on GitHub will always get pinged for this :)

You can use `Ansible Core Team` instead. Also you should add yourself to the list.

Reply:
This is in fact a copy of the sudo module. I proposed it for core as IMO it fits better together with sudo and su plugins, but alas...